Core/src/main/java/com/plotsquared/core/util/placeholders/PlaceholderRegistry.java

```diff
@@ -0,0 +1,230 @@
+        for (final PlotFlag<?, ?> flag : globalFlagContainer.getRecognizedPlotFlags()) {
+            this.registerPlaceholder(new PlotFlagPlaceholder(flag, true));
+            this.registerPlaceholder(new PlotFlagPlaceholder(flag, false));
+        }
+        GlobalFlagContainer.getInstance().subscribe((flag, type) -> {
+            this.registerPlaceholder(new PlotFlagPlaceholder(flag, true));
+            this.registerPlaceholder(new PlotFlagPlaceholder(flag, false));
+        });
+        this.createPlaceholder("currentplot_world", player -> player.getLocation().getWorld());
+        this.createPlaceholder("has_plot", player -> player.getPlotCount() > 0 ? "true" : "false");
+        this.createPlaceholder("allowed_plot_count", player -> Integer.toString(player.getAllowedPlots()));
+        this.createPlaceholder("plot_count", player -> Integer.toString(player.getPlotCount()));
+        this.createPlaceholder("currentplot_alias", (player, plot) -> plot.getAlias());
+        this.createPlaceholder("currentplot_owner", (player, plot) -> {
+            final UUID plotOwner = plot.getOwnerAbs();
+            if (plotOwner == null) {
+                return "";
+            }
+
+            try {
+                return MainUtil.getName(plotOwner, false);
+            } catch (final Exception ignored) {}
+
+            return "unknown";
+        });
+        this.createPlaceholder("currentplot_members", (player, plot) -> {
+            if (plot.getMembers() == null && plot.getTrusted() == null) {
+                return "0";
+            }
+            return String.valueOf(plot.getMembers().size() + plot.getTrusted().size());
+        });
+        this.createPlaceholder("currentplot_members_added", (player, plot) -> {
+            if (plot.getMembers() == null) {
+                return "0";
+            }
+            return String.valueOf(plot.getMembers().size());
+        });
+        this.createPlaceholder("currentplot_members_trusted", (player, plot) -> {
+            if (plot.getTrusted() == null) {
+                return "0";
+            }
+            return String.valueOf(plot.getTrusted().size());
+        });
+        this.createPlaceholder("currentplot_members_denied", (player, plot) -> {
+            if (plot.getDenied() == null) {
+                return "0";
+            }
+            return String.valueOf(plot.getDenied().size());
+        });
+        this.createPlaceholder("has_build_rights", (player, plot) ->
+            plot.isAdded(player.getUUID()) ? "true" : "false");
+        this.createPlaceholder("currentplot_x", (player, plot) -> Integer.toString(plot.getId().getX()));
+        this.createPlaceholder("currentplot_y", (player, plot) -> Integer.toString(plot.getId().getY()));
+        this.createPlaceholder("currentplot_xy", (player, plot) -> plot.getId().toString());
+        this.createPlaceholder("currentplot_rating", (player, plot) -> Double.toString(plot.getAverageRating()));
```

Comment:
Does the rating placeholder work when you are using the like system?

Reply:
yes, uses the same system internally